docs/src/main/paradox/server/walkthrough.md

```diff
@@ -200,9 +200,7 @@ In the example this was done from the `main` method, but you could also do this
 
 The above example does not use TLS and is configured to only serve HTTP/2. 
 To allow HTTP and HTTP/2 and gRPC on the same port TLS must be used.
```

Comment:
Wait, this is no longer true... perhaps this whole paragraph needs rewriting? (though linking to the TLS deploy docs from here still seems like a good idea